### PR TITLE
Functional tests

### PR DESCRIPTION
**Pull request by jvarlamova**
Add functional tests (without Travis-CI job for these tests, it will be done later).

### CODE REVIEW

File: conf.ini [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[test]
```
Comment (sc68cal) [inline conf.ini]: why is this section named test? You should probably name this section "openstack" since it involves openstack.

**Comment by sc68cal** [comment]
This will need a change to .travis.yml to actually run this new job definition, before this can be merged.

File: conf.ini (hunk 1 above)
Comment (jvarlamova) [reply conf.ini]: OK, "openstack" would be better, I'll fix it

**Comment by jvarlamova** [comment]
Work in progress

File: .travis.yml [hunk 2]
```diff
@@ -4,6 +4,7 @@ install:
 - pip install tox
 script:
 - tox
+- tox -e functional
```
Comment (sc68cal) [inline .travis.yml]: Not required. Tox by default will run anything in the envlist, so you just need to add it to 

https://github.com/sc68cal/Kostyor-cli/blob/master/tox.ini#L2

**Comment by sc68cal** [comment]
Just need to add `functional` to tox.ini and then I think this is 👍 for merge

**Pull request merged**